cmake/Platform.cmake

```diff
@@ -22,7 +22,7 @@ endif()
 if(ANDROID_ABI)
   set(ANDROID_BUILD_TOOLS_VERSION "34.0.0" CACHE STRING "Android build-tools version")
   set(ANDROID_TARGET_PLATFORM 34 CACHE INT "Target Android platform SDK version")
-  set(ANDROID_PLATFORM 23 CACHE INT "Minimum Android platform SDK version")
-  set(ANDROID_PLATFORM_INT 23 CACHE INT "Minimum Android platform SDK version") # Used in build.gradle.in
+  set(ANDROID_PLATFORM 26 CACHE INT "Minimum Android platform SDK version")
+  set(ANDROID_PLATFORM_INT 26 CACHE INT "Minimum Android platform SDK version") # Used in build.gradle.in
```

Comment:
Can we use `ANDROID_PLATFORM` in build.gradle.in?

Reply:
It wasn't possible a few years ago as far value was being modified and added some string suffix